pkgs/development/ruby-modules/gem-config/default.nix

```diff
@@ -211,12 +211,12 @@ in
     '';
   };
 
-  pg_query = attrs: lib.optionalAttrs (attrs.version == "1.3.0") {
+  pg_query = attrs: lib.optionalAttrs (attrs.version == "2.0.1") {
```

Comment:
This breaks gitlab.

@fpletz @globin @krav @talyz

Reply:
Ah, this is surprising, because just above this in the file is a "gitlab-pg_query" gem config, which I deliberately didn't touch. Perhaps that one is not actually used?

Reply:
I've pushed a version which selects either the old or new libpg_query according to the gem version: this should fix the gitlab build, but I'm on darwin so can't easily test it locally.

There's still an open question about whether the gitlab-pg_query definition is still needed.